pkg/gui/controllers/confirmation_controller.go

```diff
@@ -39,6 +41,14 @@ func (self *ConfirmationController) GetKeybindings(opts types.KeybindingsOpts) [
 			Key: opts.GetKey(opts.Config.Universal.TogglePanel),
 			Handler: func() error {
 				if len(self.c.Contexts().Suggestions.State.Suggestions) > 0 {
+					subtitle := ""
+					if self.c.State().GetRepoState().GetCurrentPopupOpts().HandleDeleteSuggestion != nil {
+						// We assume that whenever things are deletable, they
+						// are also editable, so we show both keybindings
+						subtitle = fmt.Sprintf(self.c.Tr.SuggestionsSubtitle,
+							self.c.UserConfig.Keybinding.Universal.Remove, self.c.UserConfig.Keybinding.Universal.Edit)
+					}
```

Comment:
@jesseduffield I'm showing the keybindings when the suggestions panel gets the focus (and remove it again in the opposite handler in SuggestionsController). I would have preferred not to do this here, but in GetOnFocus/GetOnFocusLost, so that it is reliably called no matter how we switch focus (maybe we want to make it possible some day to click on a suggestion to select it).

But that's not possible right now because `ContextMgr.Replace` only calls Activate on the new context, but not deactivate on the old one. I tried to "fix" that, but that's not possible because the GetOnFocusLost of both ConfirmationController and SuggestionsController destroys the panel.

I'm not sure how to solve this cleanly; let me know if you have any suggestions. For now we just put it here in the tab handler, which works fine.

Reply:
Yeah I am not sure how to solve that cleanly. Happy to use your stopgap solution